Sources/ScipioKit/Producer/PIF/PIFCompiler.swift

```diff
@@ -78,11 +80,17 @@ struct PIFCompiler: Compiler {
             )
 
             do {
-                try await xcBuildClient.buildFramework(
+                let frameworkBundlePath = try await xcBuildClient.buildFramework(
                     sdk: sdk,
                     pifPath: pifPath,
                     buildParametersPath: buildParametersPath
                 )
+
+                if buildOptions.stripDWARFSymbols {
+                    logger.info("🐛Stripping debug symbols")
+                    let binaryPath = frameworkBundlePath.appending(component: buildProduct.target.c99name)
```

Comment:
> `buildProduct.target.c99name`

This is the same as https://github.com/giginet/Scipio/blob/e678d7e3b9b6e354457716c4a184138681c36ba4/Sources/ScipioKit/Producer/PIF/FrameworkComponentsCollector.swift#L74-L82 so it's okay 🙆